rundeck_action/action_types/lib/pagerduty.ts

```diff
@@ -0,0 +1,64 @@
+import axios, { AxiosError, AxiosResponse } from 'axios';
+import { isOk, promiseResult } from '../../../../x-pack/legacy/plugins/actions/server/builtin_action_types/lib/result_type';
+import { Logger } from '../../logger'
+
+export interface PagerDutyApiOptions {
+  actionId: string;
+  url: string;
+  method: string;
+  headers: any;
+  dedupKey: string;
+  data: object;
```

Comment:
```suggestion
  data: any;
```

It doesn't look like we're actually using this as an object. Do we actually intend to enforce this as a non-primitive type?